presto-main/src/test/java/com/facebook/presto/block/BlockAssertions.java

```diff
@@ -115,6 +140,14 @@ public static Block createStringsBlock(Iterable<String> values)
         return builder.build();
     }
 
+    public static Block createStringsBlock(int positionCount, boolean allowNulls, int maxStringLength)
```

Comment:
@arhimondr Thanks for the suggestion. createRandomStringBlock does sounds more explanatory, but all the names come from the overload methods and they group together nicely. How strong do you like the names get changed?

Reply:
It's just not clear what type of block this method create. And since the data is random - one should be careful using it, as the results might not be deterministic. So i would prefer to have this contract explicitly in the name.

Reply:
Updated all methods from createXXXBlock to createRandomXXXBlock.